src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,110 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
 public class ArrayList<T> implements List<T> {
+    private static final double INCREASING_ARRAY = 1.5;
+    private static final int INITIAL_CAPACITY = 10;
+    private T[] defaultArray;
+    private int size = 0;
```

Comment:
0 is a default value
```suggestion
    private int size;
```